src/curated-corpus/components/ScheduleDayData/ScheduleDayData.tsx

```diff
@@ -0,0 +1,218 @@
+import React, { ReactElement, useState } from 'react';
+import { Box, Grid } from '@mui/material';
+import {
+  ScheduledCorpusItem,
+  ScheduledCorpusItemsResult,
+  ScheduledItemSource,
+  useRescheduleScheduledCorpusItemMutation,
+} from '../../../api/generatedTypes';
+import {
+  ScheduleDayFilterOptions,
+  ScheduleDayHeading,
+  ScheduledItemCardWrapper,
+} from '../';
+import { getDisplayTopic } from '../../helpers/topics';
+import { useNotifications, useRunMutation } from '../../../_shared/hooks';
+
+interface ScheduleDayDataProps {
+  /**
+   * The GUID of the current surface, e.g. "NEW_TAB_EN_US"
+   */
+  currentScheduledSurfaceGuid: string;
+
+  /**
+   * Data for a given date.
+   */
+  data: ScheduledCorpusItemsResult;
+
+  /**
+   * Actions to run when users click on the "Add Item" button
+   */
+  onAddItem: (date: string) => void;
+
+  /**
+   * A function that triggers a new API call to refetch the data for a given
+   * query. Needed on the Schedule page to refresh data after every action.
+   */
+  refetch?: VoidFunction;
+
+  /**
+   * Callback to set the current item (to work on, e.g. edit or reschedule)
+   * on the Schedule Page.
+   */
+  setCurrentItem: React.Dispatch<
+    React.SetStateAction<Omit<ScheduledCorpusItem, '__typename'> | undefined>
+  >;
+
+  /**
+   * A toggle function for the "Edit this item" modal.
+   */
+  toggleEditModal: VoidFunction;
+
+  /**
+   * A toggle function for the "Remove this item" modal.
+   */
+  toggleRemoveModal: VoidFunction;
+
+  /**
+   * A toggle function for the "Schedule this item" modal.
+   */
+  toggleScheduleItemModal: VoidFunction;
+
+  /**
+   * A toggle function for the "Reject and unschedule this item" modal.
+   */
+  toggleRejectAndUnscheduleModal: VoidFunction;
+}
+
+/**
+ * Show all scheduled entries for a given date.
+ *
+ * @param props
+ * @constructor
+ */
+export const ScheduleDayData: React.FC<ScheduleDayDataProps> = (
+  props,
+): ReactElement => {
```

Comment:
This component is a big chunk of functionality from SchedulePage, now in its own component. It's still too cumbersome to test effectively - I think it needs to be broken up further for that. The other three new components have test coverage.